dev/com.ibm.ws.security.fat.common.SSO.clientTests/fat/src/com/ibm/ws/security/SSO/clientTests/PrivateKeyJwt/PrivateKeyJwtClientTests.java

```diff
@@ -469,4 +484,127 @@ public void PrivateKeyJwtClientTests_noClientSSLRef() throws Exception {
 
     }
 
+    /**
+     * Test with the client config using the signature algorithm of RS256, and a private key jwt configured with RS256
+     * tokenEndpointAuthMethod="private_key_jwt"
+     * tokenEndpointAuthSigningAlgorithm="RS256"
+     * keyAliasName="altrs256"
+     * The client's alias name used with the keystore is different than the alias in the trust store
```

Comment:
```suggestion
     * The certificate's alias name used with the keystore is different than the alias in the trust store
```
I think you mean "certificate" here?

Reply:
Updated the comment